pandas/core/arrays/base.py

```diff
@@ -862,6 +863,29 @@ def copy(self) -> ABCExtensionArray:
         """
         raise AbstractMethodError(self)
 
+    def view(self, dtype=None) -> ABCExtensionArray:
+        """
+        Return a view on the array.
+
+        Parameters
+        ----------
+        dtype : str, np.dtype, or ExtensionDtype, optional
+            Default None
+
+        Returns
+        -------
+        ExtensionArray
+
+        Notes
+        -----
+        - This must return a *new* object, not self.
```

Comment:
maybe add something like "but still reference the same data"

Reply:
good idea

Reply:
will also moves Notes section to comments.  would something like a Dev Notes section in the docstring make sense (not in this PR, just brainstorming)